### PR TITLE
feat: Developer test and user instance pages

### LINKED ISSUE #583: Developer test and user instance pages
Detail see [design](https://zpl.io/VDqpzXO)

### PR DESCRIPTION
**Pull request by whDongRui**
Refactor table component style for new design
Resolve #583

### CODE REVIEW

File: src/components/Layout/index.scss [hunk 1]
```diff
@@ -14,7 +14,7 @@
   &.hasNav {
     margin: 0 20px 32px $menu-width + 20px;
     //min-width: 950px;
-    min-width: 992px;
+    min-width: calc(#{$page-min-width} - #{$menu-nav-width} - 40px);
```
Comment (iwisunny) [inline src/components/Layout/index.scss]: if `page-min-width`, `menu-nav-width` is static, no need to calc every time.

File: src/components/Toolbar/index.jsx [hunk 2]
```diff
@@ -109,11 +109,11 @@ class Toolbar extends React.Component {
           </Fragment>
         )}
         {children}
-        {!noRefreshBtn && (
+        {/* {!noRefreshBtn && (
```
Comment (iwisunny) [inline src/components/Toolbar/index.jsx]: why comment this block ?

File: src/locales/zh/apps.json [hunk 3]
```diff
@@ -146,5 +147,12 @@
   "optional": "可选",
   "fileConfig": "配置文件",
   "setPrice": "定价",
-  "updateLog": "更新日志"
+  "updateLog": "更新日志",
+
+  "Test Runtime": "测试环境",
+  "Instance Name ID": "实例名称 ID",
+  "Create Time": "创建时间",
+  "": "",
```
Comment (iwisunny) [inline src/locales/zh/apps.json]: Remove these useless lines

File: src/locales/zh/translation.json [hunk 4]
```diff
@@ -407,13 +407,19 @@
...
   "Change Password": "修改密码",
   "Notice settings": "通知设置",
   "Payment": "支付",
 
+  "Sandbox-Instances": "沙盒-实例",
+  "Customer-Instances": "用户-实例",
+  "Sandbox-Instance detail": "沙盒-实例详情",
+  "Customer-Instance detail": "用户-实例详情",
+
+
```
Comment (iwisunny) [inline src/locales/zh/translation.json]: Just keep one blank line

File: src/pages/Dashboard/Clusters/index.jsx [hunk 5]
```diff
@@ -212,19 +227,17 @@ export default class Clusters extends Component {
...
         <Toolbar noRefreshBtn noSearchBox>
-          <Button
-            type="delete"
-            onClick={() => this.operateSelected('delete')}
-            className="btn-handle"
-          >
-            {t('Delete')}
-          </Button>
           <Button type="default" onClick={() => this.operateSelected('start')}>
+            <Icon name="start" size={20} type="dark" />
```
Comment (iwisunny) [inline src/pages/Dashboard/Clusters/index.jsx]: name is a typo?

File: src/pages/Dashboard/Clusters/index.jsx [hunk 6]
```diff
@@ -212,19 +227,17 @@ export default class Clusters extends Component {
...
             {t('Start')}
           </Button>
-          <Button type="delete" onClick={() => this.operateSelected('stop')}>
+          <Button type="default" onClick={() => this.operateSelected('stop')}>
+            <Icon name="stop" size={20} type="stop" />
```
Comment (iwisunny) [inline src/pages/Dashboard/Clusters/index.jsx]: name a typo 😢

File: src/components/Toolbar/index.jsx (hunk 2 above)
Comment (whDongRui) [reply src/components/Toolbar/index.jsx]: New design has not refresh button in table

File: src/locales/zh/apps.json (hunk 3 above)
Comment (whDongRui) [reply src/locales/zh/apps.json]: ok.

File: src/locales/zh/translation.json (hunk 4 above)
Comment (whDongRui) [reply src/locales/zh/translation.json]: ok

File: src/pages/Dashboard/Clusters/index.jsx (hunk 6 above)
Comment (whDongRui) [reply src/pages/Dashboard/Clusters/index.jsx]: Console is error becase  type is error.

**Pull request merged**